src/PasswordResolver.php

```diff
@@ -17,6 +22,8 @@ public function resolve(string $secret, bool $refresh)
         try {
             $result = $this->client->post("http://localhost:8015/cache?name=$secret&refresh=$refresh");
         } catch (Throwable $e) {
+            $this->logger->error('Failed to retrieve password from cache server. ' . $e);
```

Comment:
```suggestion
            $this->logger->error('Failed to retrieve password from cache server. ' . $e->getMessage());
```

Reply:
We are relying on `__toString()` here to also have the stacktrace in the log message